### ISSUE TITLE
data management

### ISSUE THREAD

**Issue by elliotresearcher** [McSpooder/camull_net#2]
Hi,
   Nice work and I highly appreciate it. Can you please guide about how to place data in directory if someone doesn't have access to ADNI and wants to use their own dataset, and also is it possible to just use MRIs, no clinical data.
I am waiting for your kind reply.
Thanks

**Comment by McSpooder** [comment]
Hi Elliot,

I just pushed a commit to make the data directory structure more straight forward. The data should be in a directory above the project. The folder should be called data and it must contain a clinical.csv file and 4 sub-folders named after each class. There are 4 classes in total: AD, CN, pMCI and sMCI. 

[Directory visualization](https://i.gyazo.com/12705d0528b97302659a43763d4f50d4.png)

The MRI files themselves must be in the nifti format(.nii) and must be of the 110x110x110 dimension. Different dimensions can be supported but you would need to change the neural net class declaration.

It is definitely possible to run the code just with the MRI data but you will need to change the neural net class declaration in architecture.py and change the loader_helper.py and data_declaration.py scripts.

**Comment by elliotresearcher** [comment]
thanks a lot for the detailed help. it helped a lot. just a last question. 
for this line of code
    model = load_model("camull", "../weights/CN_v_AD/c51bf83c4455416e8bc8b1ebbc8b75ca/fold_5_weights-2020-04-29_13:17:33")

it gives an error as this file is not available, can you plz provide this file?

**Comment by McSpooder** [comment]
The weight files for this project are very large so I haven't included them in the Repo. You should really train the model yourself for the CN vs AD task so that it generates weights. Then use those weights to load the model and train in on the sMCI vs pMCI task. 

However, if you do insist on using my weights for the initial task here you go: https://drive.google.com/file/d/1xG1BKcEWFcC9J4-oPPByi8MCJszSEX0D/view?usp=sharing

**Comment by McSpooder** [comment]
Also for anyone else interested heres a link to all the weights that were generated for both tasks: https://drive.google.com/drive/folders/1bXztYTsxZr9MLKSbznnZiUrZruhCVnpb?usp=sharing

**Comment by elliotresearcher** [comment]
really appreciated.
 can you please also guide little about how to just use MRIs, as i don't have adni dataset, so i want to use my own dataset and i only have MRIs(no clinical data). little help will give me a path to work as I am new in python.

**Comment by McSpooder** [comment]
Sorry but thats not so straight forward. You need to make changes to the architecture.py file. Specifically to the forward function so that it only takes one input. Thats where I would start. However if you run the code with that change you will get errors from the dataloaders saying that a csv file is missing. You could just make a dummy one containing fake files to bypass it temporarily. That's as far as I'll say. DM me on twitter in a week if you still haven't figured out how to do it, I'll guide you further. I'm gonna close this issue now though.

**Comment by wolaidajiangyou** [comment]
Hello, I want to ask about SMCI and pMCI. You say it can be observed through the CSV. But the ADNI dataset I downloaded is still MCI no matter how many years later. So can you tell me how to get the data of SMCI and pMCI？

**Comment by McSpooder** [comment]
You need to find the longitudinal data on the adni site. Some subjects have multiple records. Some of these subjects convert from MCI to AD and some don't. For those that do you should update a seperate csv file and change their label to sMCI and those that don't change it to pMCI. I chose 3 years from the initial scan to determine whether they convert.

**Comment by McSpooder** [comment]
> Hello, I want to ask about SMCI and pMCI. You say it can be observed through the CSV. But the ADNI dataset I downloaded is still MCI no matter how many years later. So can you tell me how to get the data of SMCI and pMCI？

I've added a link to download the data to the readme so now you don't even need to get your own data and do data cleaning.
